src/pages/steps/modals/PictureUploadModal.tsx

```diff
@@ -243,6 +243,15 @@ const PictureUploadModal = ({
 
   const registerDescriptionProps = register('description');
 
+  const fileError =
+    errors.file?.type === 'size'
+      ? t(`pictures.upload_modal.validation_messages.file.size`, {
+          maxFileSize: MAX_FILE_SIZE / 1_000_000,
+        })
+      : errors.file?.type
+      ? t(`pictures.upload_modal.validation_messages.file.${errors.file?.type}`)
+      : undefined;
```

Comment:
Not sure the nested ternary is necessary here since you could pass the placeholders to the translation function and it'd ignore it for the other usecases no? Then you can always use the template string version

Reply:
Yes good suggestion, never thought of it this way 👍